src/App.jsx

```diff
@@ -1,14 +1,45 @@
+import { useState } from 'react';
 import './App.css';
+import messages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+
 
 const App = () => {
+
+  //set the initial state for the msgs
+  const [entries, setEntries] = useState(messages);
+
+  const getLikedCount = () => {
+    return entries.filter(entry => entry.liked).length;
```

Comment:
You can also use the `reduce` function to calculate how many likes there are from all the `entries`.